README.md

```diff
@@ -153,10 +153,10 @@ Bundlers like webpack, Rollup or browserify automatically recognize how to handl
 
 ```js
 import prettier from "prettier/standalone";
-import phpPlugin from "@prettier/plugin-php/standalone";
+import * as prettierPluginPhp from "@prettier/plugin-php/standalone";
```

Comment:
I just noticed this is for standalone version, not sure if the namespace import will work, but the entry is definitely won't work, I think we should add `./standalone` export to make it work.

@loilo WDYT?

Reply:
Good catch. What's the `./package.json` entry for, is it needed in some way? (Sorry for uneducated questions, I don't have a lot of insight into the inner workings of Prettier and its plugins, as a maintainer I'm mostly responsible for the playground. 😅)

Reply:
Maybe someone what to check `version`? 😄  Not sure what for, but lots of packages export `./package.json`.